### ISSUE TITLE
Snapping option for Georeferencing Vectors

### ISSUE THREAD

**Issue by MyGIS** [qgis/QGIS#52306]
### Feature description

It would be good to add a vertex snapping option to the georeferecing tool/window, specifically when we want to use it for vectors.

**Comment by forestcomber** [comment]
Yes, this would be very helpful.

**Comment by logan001** [comment]
yes. i really want to scale and rotate some shp files  based on another vector without applying any sort of distortion.

**Comment by albertograva** [comment]
The snap mode for vector georeferencing is really necessary (the basic options would be enough: vertex, segment, etc...).

**Comment by pigreco** [comment]
The best thing to do is a crowdfunding campaign, otherwise it will be difficult for any developer to add the requested functionality.

**Comment by albertograva** [comment]
something is moving here...[#60156](https://github.com/qgis/QGIS/pull/60156 )

**Comment by pigreco** [comment]
shall we close issue?
